astropy/units/function/magnitude_zero_points.py

```diff
@@ -1,53 +1,16 @@
-# -*- coding: utf-8 -*-
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Copy-and-paste error (I actually moved the old file and then added a new one here so that `git blame` would continue to work on the new file), will fix.